Engine/Component/ComponentLight.h

```diff
@@ -27,9 +27,8 @@ class ComponentLight : public Component
 		public: 
 			void ChangePointLightAttenuationValues(float range)
 			{
-				float augmented_range = 10 * range;
-				linear = 4.5 / (10 * augmented_range);
-				quadratic = 75.0 / (augmented_range*augmented_range);
+				linear = 4.5 / (10 * range);
+				quadratic = 75.0 / (range * range);
```

Comment:
why this values? maybe make them as a macro?

Reply:
I don't know what is this @mariofv

Reply:
Yeah, a macro would be the correct implementation. These numbers are defined [here](http://wiki.ogre3d.org/Light+Attenuation+Shortcut).